README.md

```diff
@@ -68,6 +68,7 @@ Follow these steps to make changes and release a new binary.
 6. Images will be available in the repo registry.k8s.io/dns/. The node-cache image with tag 1.15.14 can be found at registry.k8s.io/dns/k8s-dns-node-cache:1.15.14. Older versions are at registry.k8s.io/k8s-dns-node-cache:<TAG>
 7. Submit a PR for the kubernetes/kubernetes repository to switch to the new
    version of the containers. Example - https://github.com/kubernetes/kubernetes/pull/106189
```

Comment:
nit: Can we test the new descriptors in the empty kind cluster as part of this point (`kubectl apply -f ...`)? To verify they are working and no other changes apart from image version bumps are necessary in the descriptors.

Reply:
Wouldn't this manual test be equivalent to the new presubmits [suggested](https://github.com/kubernetes/dns/pull/664#issuecomment-2545524298) by Antonio? If so, then the automated approach feels much less-effort and more fail-proof to me.

Reply:
ack. I wasn't aware of the automated presubmits PR. I agree that it's much better. Thanks.